packages/trip-form/src/util.tsx

```diff
@@ -364,7 +366,7 @@ export function getCompaniesOptions(
       selected: selectedCompanies.includes(comp.id),
       text: (
         <span>
-          {CompanyIcon && <CompanyIcon />} {comp.label}
```

Comment:
This line is really long- is eslint not working on this file?

Reply:
Something is broken right? Why did that not trigger